rest_api/src/main/resources/static/assets/js/profile.js

```diff
@@ -0,0 +1,89 @@
+var profile = {};
+
+window.onload = function() {
+    profile["type"] = "tmp";//localStorage.getItem("type");
```

Comment:
Is this supposed to be like this?

Reply:
This code actually is not mine. I had to merge my branch with another one to obtain the most recent web app. But I will correct it.